applications/GeoMechanicsApplication/custom_elements/updated_lagrangian_U_Pw_diff_order_element.cpp

```diff
@@ -139,7 +139,7 @@ void UpdatedLagrangianUPwDiffOrderElement::CalculateAndAddGeometricStiffnessMatr
     MathUtils<double>::ExpandAndAddReducedMatrix(UMatrix, ReducedKgMatrix, Dim);
 
     // Distribute stiffness block matrix into the elemental matrix
-    this->AssembleUBlockMatrix(rLeftHandSideMatrix, UMatrix);
+    GeoElementUtilities::AssembleUUBlockMatrix(rLeftHandSideMatrix, UMatrix);

```

Comment:
I'm happy with the name AssembleUUBlockMatrix. Maybe we should consistently rename UMatrix to UUMatrix etc.

Reply:
I totally agree with your suggestion. Done.